docs/docker.md

```diff
@@ -51,3 +51,31 @@ Tips & Tricks
 * If you change `requirements.txt` to add dependencies for Django, you must rebuild `server`:
 
   `make build`
+
+
+Getting Production Data
+=======================
+
+Sometimes it is useful to develop against production data. To do this we
+trigger a backup on Heroku, download the backup, and import it into our docker
+conainter's database.
+
+* Create a new backup. Take note of the backup ID:
+
+  `heroku pg:backups:capture --remote prod`
```

Comment:
Might just be worth to note that prod is a named git remote.

Reply:
Good idea. I added a note about that. Thanks!